### ISSUE TITLE
textsum: resume training error

### ISSUE THREAD

**Issue by puppet101** [tensorflow/models#424]
I use the code "saver.restore(sess, checkpoint_path)" to resume training, but there is a problem when I
observe the loss curve using the tensorboard.  It seems that the loss will be reset every some fixed iterations( in my case is 500), and I have checked that the restored model weights  are correct.
Can some on help me about this issue? Thank you~
![20160919142445](https://cloud.githubusercontent.com/assets/9941762/18624052/961d2270-7e76-11e6-993c-b57f47891a66.png)

**Comment by jmchen-g** [comment]
Does the sudden change of loss happen only when you restore a model? The restore can only restore from a previous ckpt. If you reached step 1000 but only checkpointed at step 600, then your restore will go back to step 600 with worse loss.

**Comment by puppet101** [comment]
The loss did not change so suddenly in previous training, when restoring the model from the last ckpt, the loss change just like the figure...... I didn't change any parameter, and it is said that the Supervisor can restore the model automatically according to the tensorflow document.

**Comment by tatatodd** [comment]
@puppet101 are you still having this problem?

If so, can you explain your last message?  The most likely explanation for that loss shape is if each time you restore, you're restoring to an old checkpoint.  That's what @jmchen-g was hypothesizing too.  I'd suggest looking into whether that is happening.

**Comment by aselle** [comment]
Closing automatically due to lack of recent activity. Please reopen when further information becomes available. Thank you.

**Comment by dk1027** [comment]
@puppet101 are you able to resolve this problem? I am having a similar problem where the loss is reset each time I restore from a checkpoint. It only happens if I restart the script (or in the case of jupyter notebook, restart the kernel).
